packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -176,6 +180,14 @@ const ActivityDetailsSidebar = (props: Props) => {
     </div>
   )
 
+  const handleUpgrade = () => {
+    SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
```

Comment:
We also want to have a `Upgrade CTA Viewed` event to complete the funnel

Reply:
@tianrunhe In which cases do we want to track the CTA impression? I.e. once per page, every time the CTA mounts, etc.

If a user is a member of some paid teams and some free teams, switching between paid+free in a team picker will lead to the CTA mounting each time a free team is selected, which seems like it could skew event-level funnel results if we track this every time it mounts.

Reply:
OK yeah that make sense. I checked again, we don't have `Upgrade CTA Viewed` events for `teamTemplate` location today either. All good then!